scripts/release-docker.sh

```diff
@@ -90,20 +98,19 @@ else
   REPO="--build-arg MINA_REPO=${BUILDKITE_PULL_REQUEST_REPO}"
 fi
 
+tag="gcr.io/o1labs-192920/$SERVICE:$VERSION"
```

Comment:
NIT: can you make this ALL_CAPS like the other variables in this file (and same with target below)